### PR TITLE
Readme improvements

### LINKED ISSUE #39: Trim Readme to keep relevant content
Move content from the readme that discusses how to contribute, and place it in a separate file in the docs folder / or put it as CONTRIBUTING GUIDELINES (check, GitHub has something specific for that).

### PR DESCRIPTION
**Pull request by xNetcat**
Closes #39 #38 #36

### CODE REVIEW

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+* Fork the repository by clicking the fork button on top right corner of the page
+* Clone the target repository. To clone, click on the clone button and copy the https address. Then run
+
+`git clone [HTTPS-ADDRESS]`
+
+* Go to the cloned directory by running
+
+`cd [NAME-OF-REPO]`
+
```
Comment (YashKumarVerma) [inline PULL_REQUEST_TEMPLATE.md:11]: PR template should not have the method to create a PR, rather fields to fill while sending PR.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Guidelines for Pull Request
+
+* Avoid pull requests that :
+      *are automated or scripted
+      * that are plagarized from someone else's branch
+  * Do not spam
+  * Project maintainer's decision on validity of PR is final.
+
+  For additional guidelines, refer to [participation rules](https://hacktoberfest.digitalocean.com/details#rules)
+
```
Comment (YashKumarVerma) [inline CONTRIBUTING.md:10]: these rules are added, move the PR related content from readme to docs folder.

**Review by YashKumarVerma — CHANGES_REQUESTED** [review]

File: PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (xNetcat) [reply PULL_REQUEST_TEMPLATE.md]: ff492c90efc51e2839af7cbe0a4ecd7ae1bb141e fixes the pr template

File: CONTRIBUTING.md (hunk 2 above)
Comment (xNetcat) [reply CONTRIBUTING.md]: d2f8994 not sure if this is what you meant by moving the content to docs

**Review by YashKumarVerma — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -7,62 +7,28 @@
...
-    <img alt="Github Actions" src="https://github.com/ACM-VIT/dolos/workflows/Node.js%20CI/badge.svg" />
-</p>
-<img alt="GitHub Hacktoberfest combined status" src="https://img.shields.io/github/hacktoberfest/2020/ACM-VIT/dolos?style=for-the-badge">
+[![made-by-acm](https://img.shields.io/badge/MADE%20BY-ACM%20VIT-blue?style=for-the-badge)](https://acmvit.in/)
+![GitHub last commit](https://img.shields.io/github/last-commit/ACM-VIT/dolos?style=for-the-badge)
+![GitHub repo size](https://img.shields.io/github/repo-size/ACM-VIT/dolos?style=for-the-badge)
+[![License](https://img.shields.io/badge/License-MIT-green.svg?style=for-the-badge)](https://en.wikipedia.org/wiki/MIT_License)
+[![GitHub forks](https://img.shields.io/github/forks/ACM-VIT/dolos?style=for-the-badge)](https://github.com/ACM-VIT/dolos/network)
+[![GitHub stars](https://img.shields.io/github/stars/ACM-VIT/dolos?style=for-the-badge)](https://github.com/ACM-VIT/dolos/stargazers)
+![Github Actions](https://github.com/ACM-VIT/dolos/workflows/Node.js%20CI/badge.svg)
```
Comment (jmmendivil) [inline README.md:18]: You can change the src for:

`https://img.shields.io/github/workflow/status/ACM-VIT/dolos/Node.js%20CI?style=for-the-badge`

and your PR closes #42 too

Comment (xNetcat) [reply README.md]: c4df8ed closes #42

File: README.md [hunk 4]
```diff
@@ -7,62 +7,28 @@
...
-* Push to remote. To do this, run 
-<pre><code>git push origin [YOUR-BRANCH-NAME]</code></pre>
-* Create a pull request. Go to the target repository and click on the "Compare & pull request" button. **Make sure your PR description mentions which issues you're solving.**
-<img src="https://drive.google.com/u/1/uc?id=1f9JKAR-kRvCRGxIs_SAvegaYDPx53T9G&export=download"></img>
-* Wait for your request to be accepted. 
-
----
-## Guidelines for Pull Request
-
-<!-- general guidelines here -->
```
Comment (YashKumarVerma) [inline README.md:58]: all this content needs to be in a file right ? it's moving content, not deleting it.

**Review by YashKumarVerma — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (xNetcat) [reply README.md:58]: you are right, fixing it right now

Comment (xNetcat) [reply README.md]: b60ef68 added missing content

File: CONTRIBUTING.md (hunk 2 above)
Comment (YashKumarVerma) [reply CONTRIBUTING.md]: create a new file in docs folder, which has instructions on how to contribute

File: README.md [hunk 5]
```diff
@@ -7,62 +7,49 @@
...
+<pre><code> git checkout -b [YOUR-BRANCH-NAME]</code></pre>	
+* Make your changes to the code. Add changes to your branch by using 	
+<pre><code>git add .</code></pre>	
+* Commit the chanes by executing	
+<pre><code>git commit -m "your msg"</code></pre>	
+* Push to remote. To do this, run 	
+<pre><code>git push origin [YOUR-BRANCH-NAME]</code></pre>	
+* Create a pull request. Go to the target repository and click on the "Compare & pull request" button. **Make sure your PR description mentions which issues you're solving.**	
+<img src="https://drive.google.com/u/1/uc?id=1f9JKAR-kRvCRGxIs_SAvegaYDPx53T9G&export=download"></img>	
+* Wait for your request to be accepted.
```
Comment (YashKumarVerma) [inline README.md:52]: this content needs to go to a separate file in docs folder.

**Review by YashKumarVerma — CHANGES_REQUESTED** [review]

**Review by YashKumarVerma — APPROVED** [review]

**Pull request merged**